### PR TITLE
Feature.integrationtests

### PR DESCRIPTION
**Pull request by indy-independence**
Add integrationtests.sh that runs more general system tests via API and measures code coverage. Also runs unit tests to combine results for code coverage. Docker containers has been updated to get code coverage and scheduler to work with uwsgi.

### CODE REVIEW

File: docker/api/cnaas-setup.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+# Start venv
+python3 -m venv /opt/cnaas/venv
+cd /opt/cnaas/venv/
+source bin/activate
+
+/opt/cnaas/venv/bin/pip install -U pip
+
+# Fetch the code and install dependencies
+git clone https://github.com/SUNET/cnaas-nms.git
```
Comment (krihal) [inline docker/api/cnaas-setup.sh]: Is this really a good idea? I'd like to be sure that tagged containers really runs a specific version of the code.

File: docker/api/cnaas-setup.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+set -x
+
+export DEBIAN_FRONTEND noninteractive
+
+/bin/sed -i s/deb.debian.org/ftp.se.debian.org/g /etc/apt/sources.list
```
Comment (krihal) [inline docker/api/cnaas-setup.sh]: Think you can do something like: ENV REPO ftp.se.debian.org

File: docker/api/cnaas-setup.sh (hunk 1 above)
Comment (indy-independence) [reply docker/api/cnaas-setup.sh]: I think this is the same as for other containers, cnaas-setup.sh is only run once at build time of the container. But I changed so we don't git pull on runtime (exec-pre-app.sh), that was only for development purposes

File: docker/api/cnaas-setup.sh (hunk 2 above)
Comment (indy-independence) [reply docker/api/cnaas-setup.sh]: This is something I stole from the eduid project but it seems to be working quite well. Do you know how to do it with env variables instead? I didn't find anything on a quick google search

**Comment by krihal** [comment]
I'm merging this monstrous thing. :-)

**Pull request merged**